### PR TITLE
Added Badge/Toast & fixed minor bugs

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Its a small PR. Added Toast and Badge. And fixed some minor bugs.
Please review

### CODE REVIEW

**Review by Varu98 — APPROVED** [review]

**Review by Varu98 — APPROVED** [review]
Looks good to merge, 

maybe we can remove the unused variables in the imports of files to get rid of the Netlify deployment errors

also try removing href attributes from the anchor tags as they also cause warnings further hindering the deployment process

File: src/Components/Navbar/Navbar.css [hunk 1]
```diff
@@ -1,107 +1,122 @@
...
-    z-index: 999;
-    box-shadow: 3px 3px 12px 0 rgb(0 0 0 / 15%);
+  display: flex;
+  align-items: center;
+  position: sticky;
+  top: -2px;
+  height: 4.125rem;
+  width: 100%;
+  background-color: var(--main-background-color);
+  z-index: 999;
```
Comment (ashishkatiyar2404) [inline src/Components/Navbar/Navbar.css:29]:
```suggestion
  z-index: 10;
```
can also work

File: src/Components/TotalBill/TotalBill.jsx [hunk 2]
```diff
@@ -1,9 +1,6 @@
 import React, { useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
 import { useCart } from "../../context/cart-context";
-
-// import { useWishlist } from "../../context/wishlist-context";
-
 import "./TotalBill.css";
 export function TotalBill() {
```
Comment (ashishkatiyar2404) [inline src/Components/TotalBill/TotalBill.jsx:5]: you can give space between imports files and exports

File: src/context/cart-context.js [hunk 3]
```diff
@@ -58,14 +59,14 @@ const CartProvider = ({ children }) => {
           );
           if (response.status === 201) {
             cartDispatch({ type: "ADD_TO_CART", payload: response.data.cart });
-            console.log("Product added to Cart");
+            Toast({ type: "success", message: "Product added to Cart" });
           }
         } catch (error) {
           console.log(error);
```
Comment (ashishkatiyar2404) [inline src/context/cart-context.js:65]: you can remove console logs

**Review by ashishkatiyar2404 — APPROVED** [review]
Code functionality is good, suggest some changes and pls remove the console. logs.
You can merge after few changes

**Pull request merged**